app/controllers/users_controller.rb

```diff
@@ -0,0 +1,35 @@
+class UsersController < ApplicationController
+  def edit
+    @user = User.find(params[:id])
+  end
+
+  def destroy
+    @user = User.find(params[:id])
+    @user.destroy
+    flash[:notice] = "Account successfully deleted!"
+    redirect_to root_path
+  end
+
+  def update
+    @user = User.find(params[:id])
+    @email = params[:user][:email]
+    if @user.email != @email
+      if @user.update(email: @email)
```

Comment:
redzi - šeit tev ir būtība ir tas pats kas iekš paramiem: `{user: {email: email}}`.

Mēs šo saucam par mass assignment, jo tu ērti varētu updeitot kaut 100 atribūtus nerakstot visu ar roku, ja izmantotu params.require(:user).permit({list of attributes goes here})